src/settings.c

```diff
@@ -78,7 +78,7 @@ static const GPreConfLog logs = {
   "%^ %v [%d:%t %^] %h %^\"%r\" %s %^ %b %^ %L %^ \"%R\" \"%u\"", /* Amazon S3 */
 
   /* Caddy JSON */
-  "{ \"ts\": \"%x.%^\", \"request\": { \"remote_addr\": \"%h:%^\", \"proto\":"
+  "{ \"ts\": \"%x.%^\", \"request\": { \"remote_addr\": \"%h\", \"proto\":"
```

Comment:
The field has also been renamed to `remote_ip`, and there's also a new `remote_port` as well